tasks/main.yml

```
@@ -16,6 +16,36 @@
     name: "{{ package_names }}"
     state: present
 
+- name: Configure AppArmor for Clamd
+  when: "ansible_apparmor.status == 'enabled' and apparmor_complain"
```

Remove unnecessary quotes:
```suggestion
  when: ansible_apparmor.status == "enabled" and apparmor_complain
```